client/src/templates/Challenges/utils/build.js

```diff
@@ -164,16 +165,27 @@ export function buildJSChallenge({ files }, options) {
     .map(pipeLine);
   return Promise.all(finalFiles)
     .then(checkFilesErrors)
-    .then(files => ({
-      challengeType: challengeTypes.js,
-      build: files
+    .then(files => {
+      let build = files
         .reduce(
           (body, file) => [...body, file.head, file.contents, file.tail],
           []
         )
-        .join('\n'),
-      sources: buildSourceMap(files)
-    }));
+        .join('\n');
+      let sources = buildSourceMap(files);
+      if (options?.removeComments !== false) {
+        build = curriculumHelpers.removeJSComments(build);
+        sources = {
+          ...sources,
+          index: curriculumHelpers.removeJSComments(sources.index)
```

Comment:
```suggestion
        build = curriculumHelpers.removeJSComments(build);
        sources = {
          ...sources,
          index: removeJSComments(sources.index)
```

Reply:
Ah I see, ok will do! Thank you for clarifying!